yarn-project/blockchain/src/aztec_chain.ts

```diff
@@ -0,0 +1,16 @@
+import { Chain } from 'viem';
+
+/**
+ * Interface containing the connection and chain properties to interact with a blockchain.
+ */
+export interface AztecChain {
```

Comment:
This feels ambiguous, aztec chain makes me think this is refering to the l2 rather than the l1